src/main/webapp/coffeeshop.html

```diff
@@ -9,10 +9,18 @@
     <script src="map.js"></script>
     <script src="script.js"></script>
   </head>
-  <body onload="loadStoreInfo()" id= "background">
-    <a href="index.html" class="button">Home</a>
-    <a href="user.html" class="button">User</a>
-    <a href="store.html" class="button">Stores</a>
+  <body onload="loadStoreInfo(); determineLogin(); loadRatings();" id= "background">
```

Comment:
Have we tested all 3 functions are called sequentially and work as expected?
Also, as an FYI, read about alternative approaches here: https://www.htmlgoodies.com/beyond/javascript/article.php/3724571/using-multiple-javascript-onload-functions.htm

Reply:
Also: https://stackoverflow.com/questions/33785313/javascript-loading-multiple-functions-onload

Reply:
none of the three function calls are dependent on each other, so whichever way the functions are called (as long as all of them are called) will suffice for our plans